projects/aca-playwright-shared/src/page-objects/pages/recent-files.page.ts

```diff
@@ -42,4 +42,12 @@ export class RecentFilesPage extends BasePage {
   public viewer = new ViewerComponent(this.page);
   public sidenav = new SidenavComponent(this.page);
   public breadcrumb = new Breadcrumb(this.page);
+  public infoDrawer = new AdfInfoDrawerComponent(this.page);
+
+
+  async addCommentToNode(commentText: any) {
```

Comment:
As mentioned can be moved to info drawer

Reply:
Will be fixed in the next push